### ISSUE TITLE
"$speller->checkText($source, $languages)" will cache "$languages" for all subsequent requests

### ISSUE THREAD

**Issue by Slayer911** [mekras/php-speller#51]
Hi, I found the following error:
```php
$speller =  new ExternalSpeller(...); // In the example I use an abstract class because the error is here
$speller->checkText($source, ['en']);
// the command "hunspell -i UTF-8 -a -d en_US" will be generated
$speller->checkText($source, ['uk']);
// "hunspell -i UTF-8 -a -d en_US" will also be generated and hunspell will ignore the language change
```
The problem exist in: `mekras/php-speller/src/ExternalSpeller.php` method "composeProcess":
```php
    private function composeProcess(array $command): Process
    {
        if ($this->process === null) {
            $this->process = new Process($command);
        }

        $this->process->setTimeout($this->timeout);

        return $this->process;
    }
 ```
because it caches the "process" and does not use the existing "resetProcess" method to clear them.

**Comment by icanhazstring** [comment]
Good catch :)
Will take care of that 👍

**Comment by icanhazstring** [comment]
Fixed with 2.3.0

**Comment by Slayer911** [comment]
thank you!
